terraform/prod.sh

```diff
@@ -0,0 +1 @@
+export TF_VAR_mapbox_token=pk.eyJ1IjoiYXZrb3MiLCJhIjoiY2t4bmUweGhoNWxxZjJybzVuYzlxZ2tyaCJ9.mbQbEeHNKfExq1a7kqvpTg
```

Comment:
How would you manage secrets in production?

Reply:
with secret manager from AWS